src/System.Text.Json/src/System/Text/Json/Node/JsonArray.cs

```diff
@@ -0,0 +1,675 @@
+        /// </summary>
+        /// <param name="value">The value to add.</param>
+        public void Add(int value) => Add(new JsonNumber(value));
+
+        /// <summary>
+        ///   Adds the specified value as a <see cref="JsonNumber"/> to the JSON array.
+        /// </summary>
+        /// <param name="value">The value to add.</param>
+        public void Add(long value) => Add(new JsonNumber(value));
+
+        /// <summary>
+        ///   Adds the specified value as a <see cref="JsonNumber"/> to the JSON array.
+        /// </summary>
+        /// <param name="value">The value to add.</param>
+        /// <exception cref="ArgumentException">
+        ///   Provided value is not in a legal JSON number format.
+        /// </exception>
+        public void Add(float value) => Add(new JsonNumber(value));
+
+        /// <summary>
+        ///   Adds the specified value as a <see cref="JsonNumber"/> to the JSON array.
+        /// </summary>
+        /// <param name="value">The value to add.</param>
+        /// <exception cref="ArgumentException">
+        ///   Provided value is not in a legal JSON number format.
+        /// </exception>
+        public void Add(double value) => Add(new JsonNumber(value));
+
+        /// <summary>
+        ///   Adds the specified value as a <see cref="JsonNumber"/> to the JSON array.
+        /// </summary>
+        /// <param name="value">The value to add.</param>
+        [CLSCompliant(false)]
+        public void Add(sbyte value) => Add(new JsonNumber(value));
+
+        /// <summary>
+        ///   Adds the specified value as a <see cref="JsonNumber"/> to the JSON array.
+        /// </summary>
+        /// <param name="value">The value to add.</param>
+        [CLSCompliant(false)]
+        public void Add(ushort value) => Add(new JsonNumber(value));
+
+        /// <summary>
+        ///   Adds the specified value as a <see cref="JsonNumber"/> to the JSON array.
+        /// </summary>
+        /// <param name="value">The value to add.</param>
+        [CLSCompliant(false)]
+        public void Add(uint value) => Add(new JsonNumber(value));
+
+        /// <summary>
+        ///   Adds the specified value as a <see cref="JsonNumber"/> to the JSON array.
+        /// </summary>
+        /// <param name="value">The value to add.</param>
+        [CLSCompliant(false)]
+        public void Add(ulong value) => Add(new JsonNumber(value));
+
+        /// <summary>
+        ///   Adds the specified value as a <see cref="JsonNumber"/> to the JSON array.
+        /// </summary>
+        /// <param name="value">The value to add.</param>
+        public void Add(decimal value) => Add(new JsonNumber(value));
+
+        /// <summary>
+        ///   Inserts the specified item at the specified index of the JSON array.
+        /// </summary>
+        /// <param name="index">The zero-based index at which item should be inserted.</param>
```

Comment:
```suggestion
        /// <param name="index">The zero-based index at which <paramref name="item"/> should be inserted.</param>
```

(applies to all the overloads)